### PR TITLE
added git_readme

### PR DESCRIPTION
**Pull request by Spnetic-5**
Please review!

### CODE REVIEW

**Comment by gautam-dev-maker** [comment]
* Add ```Table Of Content``` , refer [this](https://github.com/SRA-VJTI/Wall-E_v2.2/blob/master/3_MPU/README.md) readme
* Also add the content for git commands for commit ,pull & push

**Comment by Spnetic-5** [comment]
Okay!

**Comment by Spnetic-5** [comment]
Okay!

**Comment by meshtag** [comment]
Thanks @Spnetic-5 for your contribution!
Can you please add some [info](https://frontendmasters.com/courses/git-in-depth/working-area-staging-area-repository/#:~:text=The%20working%20area%20is%20where,occur%20for%20the%20next%20commit.) about working area, staging area and upstream repository before explaining/introducing "commits" to the reader.

**Comment by Spnetic-5** [comment]
@meshtag on it!

**Comment by SAtacker** [comment]
@Spnetic-5 could you add pull vs fetch (a line at least).
A general suggestion to keep filenames/folder `lower-case` names.

**Comment by Spnetic-5** [comment]
Yes @SAtacker, @kkothari2001 is working on another readme called git_commands.md, `pull vs fetch` along will all other git commands will be covered there.

**Comment by Spnetic-5** [comment]
Kindly review `task.md` and give suggestions.

**Comment by gautam-dev-maker** [comment]
# 2.git_workflow.md
* No image is visible, correct it

# 4.task.md

* Do not add process to do it, just add the task to do  , and provide reference to markdown file where they can see how to do it.

Change names of files to this format “2_git_workflow.md”, same goes for all other files

**Comment by Spnetic-5** [comment]
@gautam-dev-maker, @amanchhaparia, @meshtag

File: git_github/1_git_intro .md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+### 2. Let’s create a new public repository. You can call it whatever you like if the name is available also add a small description.
+<img src="https://github.com/Spnetic-5/Intro_to_CV/blob/main/git_github/assets/2.png" align="center">
+
+### 3. Click on Initialise repo with a README.md file, which contains information about the project. Choose a License also & Click on `Create Repository` button.
+<img src="https://github.com/Spnetic-5/Intro_to_CV/blob/main/git_github/assets/3.png" align="center">
+
+### 4. Here is how the repository should look:
+<img src="https://github.com/Spnetic-5/Intro_to_CV/blob/main/git_github/assets/4.png" align="center">
+
```
Comment (SAtacker) [inline git_github/1_git_intro .md:53]: Please fix the links i.e. keep them relative `src="./assets/1.png"`.

File: git_github/1_git_intro .md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+# Sync and interact with your repository through the command line:
+
+### Run the following command in Terminal or cmd, change `<username>` with your own username:
```
Comment (SAtacker) [inline git_github/1_git_intro .md:56]: This is just cloning. Where's syncing?
Can we replace it all by just cloning?

**Review by SAtacker — CHANGES_REQUESTED** [review]
I have left a few comments in the review.
Mostly the content seems enough.
However, I couldn't help but notice that you are always creating a repository online and then cloning and doing the operations. I think we all know that there is a better way to do this.

Lastly, there's no demo for `git-reset` , some examples showing how to fetch remote and test changes without actually creating crappy branches, etc. but that can be left for later.
Regards,
Shreyas

**Comment by kkothari2001** [comment]
@SAtacker the word "DEMO", was put just to show that there will be a demo when I explain. Concepts in git reset are difficult to explain in pictures only.

**Pull request merged**